mobile/native/src/ln_dlc/mod.rs

```diff
@@ -525,6 +532,136 @@ pub fn get_usable_channel_details() -> Result<Vec<ChannelDetails>> {
     Ok(channels)
 }
 
+/// Syncs the position with the dlc channel state.
+///
+/// - DLC Channel in state `Signed` but no position: Create position from `filling` order.
+/// - DLC Channel in state `OffChainClosed` and a position exists. Delete the position.
+/// - DLC Channel in state `CloseOffered` or `CloseAccepted`: Inform the UI that the dlc channel is
+///   recovering.
+/// - DLC Channel in state `Offered`, `Accepted` or `Finalized`: Inform the UI that the dlc channel
+///   is recovering.
+/// - DLC Channel in any other state but with position: Delete position the channel might have been
+///   force closed.
+async fn sync_position_with_dlc_channel_state() -> Result<()> {
```

Comment:
🤔 is there any way to cover this functionality with tests?

Reply:
oh yeah, I think thats a very good idea 🙈. I will try to write some tests for that! 👍

Reply:
I had a look into it and I fear adding regression tests for this functionality will bring a lot more effort and code changes to this PR than I want to invest at the moment.

I'll make sure that I have everything manually tested though.